recipes/bmi-c/build.sh

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/env bash
```

Comment:
Could you please remove this shebang?
``conda-build`` already handles it :)
```suggestion
```

Reply:
Done. Thanks, @marcelotrevisani!